extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -269,14 +189,17 @@ export class Bookend {
 
     this.isBuilt_ = true;
 
-    this.root_ = this.win_.document.createElement('div');
-    this.bookendEl_ = renderAsElement(this.win_.document, ROOT_TEMPLATE);
+    this.root_ = this.win.document.createElement('div');
+    this.bookendEl_ = renderAsElement(this.win.document, ROOT_TEMPLATE);
 
     createShadowRootWithStyle(this.root_, this.bookendEl_, CSS);
 
     this.replayButton_ = this.buildReplayButton_();
 
-    const ampdoc = getAmpdoc(this.parentEl_);
+    let ampdoc;
```

Comment:
Now that this inherits from ``AMP.BaseElement``, you can get the ``ampdoc`` using ``this.getAmpDoc()``.
This allows you to delete:
- the ``parentEl_`` property
- the ``buildCallback`` method
- and these four lines : )

Reply:
nice, thanks!